shell/platform/windows/settings_plugin_unittests.cc

```diff
@@ -70,5 +72,18 @@ TEST(SettingsPluginTest, StartWatchingStartsWatchingChanges) {
   settings_plugin.StartWatching();
 }
 
+TEST(SettingsPluginTest, HighContrastModeHonored) {
+  TestBinaryMessenger messenger([](const std::string& channel,
+                                   const uint8_t* message, size_t message_size,
+                                   BinaryReply reply) {});
+  ::testing::NiceMock<MockSettingsPlugin> settings_plugin(&messenger, nullptr);
+
+  settings_plugin.UpdateHighContrastMode(true);
+  EXPECT_TRUE(settings_plugin.is_high_contrast());
```

Comment:
Can we check this sends a message?

Reply:
I can make the unit test more robust. `SendSettingsSendsMessage` checks that the message is actually sent; I'll make sure `SendSettings` is called right.